auth/build.gradle

```diff
@@ -30,6 +30,8 @@ android {
 dependencies {
     implementation project(path: ':core')
     implementation 'net.openid:appauth'
+    //noinspection GradleCompatible
```

Comment:
What about this?
`implementation "com.android.support:customtabs:26.1.0"
implementation ('net.openid:appauth', {
       exclude group: 'com.android.support', module: 'customtabs'
})`

Reply:
@wojta This seems to work too, would it be possible to get an example of what it does/why it helps? Not too familiar with this whole ecosystem